apps/workers/db-check/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jmoiron/sqlx"
+	tools "github.com/pcelvng/task-tools"
+	"github.com/pcelvng/task-tools/bootstrap"
+	"github.com/pcelvng/task-tools/db"
+	"github.com/pcelvng/task/bus"
+)
+
+const (
+	taskType    = "task.dbcheck"
+	description = `
+Runs record count checks against a given table's data to verify that the data is updated as expected
+
+uri params:
+table      - required table name to validate
+date_field - required field name that holds the date value
+date1      - 1st date value to check for records
+date2      - 2nd date to compare row counts between date1 and date2
```

Comment:
🤔 
Is there a value to make these easier to understand maybe we could do a custom struct to get the point across something like 
`date=2022-05-17:-24h` or `date=2022-05-17:2022-06-12` or we could allow an array of dates to compare against each other so `date=2022-05-17,2022-06-12`

Reply:
This is a good thought. I like the idea of having more than just the two dates. I'll see what I can come up with.